libs/auth-api-lib/src/lib/entities/dto/base/client-base.dto.ts

```diff
@@ -280,4 +280,32 @@ export abstract class ClientBaseDTO {
     example: null,
   })
   contactEmail!: string
+
+  @IsBoolean()
+  @IsNotEmpty()
+  @ApiProperty({
+    example: true,
+  })
+  readonly supportsDelegation!: boolean
+
+  @IsBoolean()
+  @IsNotEmpty()
+  @ApiProperty({
+    example: true,
+  })
+  readonly supportsLegalGuardians!: boolean
```

Comment:
Shouldn't these be nullable?

Reply:
This is the dto. The thought being that this would be turned to non nullable in the DB and in the models after updating clients that already exist